src/Infrastructure/DependencyInjection.cs

```diff
@@ -0,0 +1,48 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity;
+
+using Codidact.Authentication.Infrastructure.Persistance;
+using Codidact.Authentication.Domain.Entities;
+
+namespace Codidact.Authentication.Infrastructure
+{
+    public static class DependencyInjection
+    {
+        public static IServiceCollection AddInfrastructure(this IServiceCollection services, IConfiguration configuration)
+        {
+            services
+                .AddDbContext<ApplicationDbContext>(options =>
+                {
+                    options.UseSqlite(configuration.GetConnectionString("Authentication"));
+                });
+
+            services
+                .AddIdentityCore<ApplicationUser>(options =>
+                {
+                    options.Password.RequireDigit = false;
```

Comment:
Note in the future we will need 1 unique, one lower one upper one digit.
For example: Aa123456!

Reply:
There was a [discussion about passwords][1]. My understanding was that we only check for the length of the password and accept the (possibly insecure) password they choose. I used eight as the minimum such that I could use 'password' for the test user...

  [1]: https://forum.codidact.org/t/mvp-account-database-scheme-suggestion/337/50?u=asynts

Reply:
Alright if it was discussed then so shall it be